Main.cpp

```diff
@@ -155,7 +155,6 @@ namespace RTE {
 	void RunMenuLoop() {
 		g_UInputMan.DisableKeys(false);
 		g_UInputMan.TrapMousePos(false);
-		g_AudioMan.StopAll();
```

Comment:
Does this not cause sound effects to bleed into the menu when pausing?

Reply:
There's ActivityMan that already does this - see ActivityMan::PauseActivity